index.html

```diff
@@ -2358,9 +2349,11 @@ <h3 id=new-session><dfn data-lt="new sessions|creating a new session">New Sessio
 variables</var> and <var>parameters</var> are:
 
 <ol>
-  <li><p>If the <a>maximum active sessions</a> is equal to
-  the length of the list of <a>active sessions</a>,
-  return <a>error</a> with <a>error code</a> <a>session not created</a>.
+  <li><p>If the implementation is an <a>endpoint node</a>, and the
+  list of <a>active HTTP sessions</a> is not empty, or otherwise if
+  the implementation is unable to start an additional session,
```

Comment:
Perfect addition given that there can be a lot of other issues during session creation.